mreg/api/v1/tests/tests.py

```diff
@@ -280,6 +283,26 @@ def test_login_with_invalid_credentials(self):
         self.assert_post_and_400("/api/token-auth/", {"who": "someone", "why": "because"})
         self.assert_post_and_400("/api/token-auth/", {})
 
+class APIMetaTestCase(MregAPITestCase):
+    """Test the meta API endpoint."""
+
+    def test_meta_versions_admin_200_ok(self):
+        response = self.assert_get_and_200("/api/meta/versions")
+        for key in ('rest_framework_version', 'django_version', 'python_version'):
+            self.assertTrue(key in response.data)
```

Comment:
Should use `with self.subTest():` when looping and asserting.

Reply:
Fixed in https://github.com/unioslo/mreg/pull/540